### ISSUE TITLE
app 多用户下载  能不能根据tag区分任务的

### ISSUE THREAD

**Issue by xiaolingcong** [lingochamp/okdownload#317]
多用户对一个文件采用的是post 下载。是怎么区分是不是同一个任务的，根据下载URL吗

**Comment by rantianhua** [comment]
对 OkDownload 来说，是没有多用户这个概念的，在构建 Task 时，需要给 Task 创建 id ，这个是根据当前 task 数量来的。当然在创建时，需要判断这个 Task 是不是和已经创建的某个 Task 是同一个 Task ，此时的判断就是依据 Task 的目标文件路径和 url 一起。

**Comment by rantianhua** [comment]
所以你可以再明确下你的问题，是否只是想知道如果利用 OkDownload 区分不同的 Task ？

**Comment by xiaolingcong** [comment]
> 所以你可以再明确下你的问题，是否只是想知道如果利用 OkDownload 区分不同的 Task ？  

   是的 因为我这是post请求文件路径和url 对于没个用户都是一样的。所以想能不能下载的时候专门，
   传一个tag 来标识是不是不同的task。 这个tag 可能是用户ID+下载目标的id 这样标识。这样下载的
   话，不同的用户下载相同的文件也不会被认为是同一个文件。 (ps,文件是加密过的所以 用户之间的 
   文件是不能通用的)

**Comment by rantianhua** [comment]
用户之间的文件是不能通用，所以你是希望不同用户下载同一个文件不会相互影响，就像在各自下载自己的文件一样？

**Comment by xiaolingcong** [comment]
对的 是这样

**Comment by rantianhua** [comment]
那每个用户建立任务时目标文件地址不同即可。

**Comment by rantianhua** [comment]
不过现在再看你提的用 tag 来区分任务，确实可以考虑下这个思路。
